### PR TITLE
Cmad 34

### PR DESCRIPTION
**Pull request by anbublacky**

### CODE REVIEW

File: lib/cm_admin/view_helpers/page_info_helper.rb [hunk 1]
```diff
@@ -5,6 +5,42 @@ def page_title
...
+            title + " create record"
+          when 'edit'
+            title + " edit record"
+          end
+        end
+      end
+
+      def action_description
+        if @model.current_action.page_title
+          title = @model.current_action.page_title
```
Comment (AdityaTiwari2102) [inline lib/cm_admin/view_helpers/page_info_helper.rb:28]: @anbublacky - This method should be for `page_description` right?
```suggestion
        if @model.current_action.page_description
          title = @model.current_action.page_description
```

File: lib/cm_admin/view_helpers/page_info_helper.rb [hunk 2]
```diff
@@ -5,6 +5,42 @@ def page_title
...
+          case action_name
+          when 'index'
+            title + " list record"
+          when 'show'
+            title + " detail"
+          when 'new'
+            title + " new record"
+          when 'edit'
+            title + " edit record"
+          end
```
Comment (AdityaTiwari2102) [inline lib/cm_admin/view_helpers/page_info_helper.rb:40]: @anbublacky - Can we have a common method/helper for the `else` case of `action_title` and `action_description` which will return the title as both have the same content?

**Review by AdityaTiwari2102 — CHANGES_REQUESTED** [review]

File: lib/cm_admin/view_helpers/page_info_helper.rb (hunk 2 above)
Comment (anbublacky) [reply lib/cm_admin/view_helpers/page_info_helper.rb:40]: @AdityaTiwari2102 I think it is better to have different method, as the content will be different. Right now i have kept the same content.

Comment (anbublacky) [reply lib/cm_admin/view_helpers/page_info_helper.rb:40]: Merging this for now. Create a ticket if needed for improvement.

**Pull request merged**